### DISCUSSION TITLE
til-site/posts/the-rule-of-four/

### DISCUSSION THREAD

**Discussion by giscus** [hiAndrewQuinn/til#5] [Announcements]
# til-site/posts/the-rule-of-four/

A communication heuristic, optimized for asynchronous communication of detailed concepts between human beings.
Not really a TIL, I’ve been sitting on this one for a while.
Rationale Existence “proof” Our short term memory can hold at best about 7 +/- 2 chunks of information at a time. Start with the lower end of that: 5 chunks. Assume that whatever someone is “really” trying to do takes up 1 of those 5.

https://hiandrewquinn.github.io/til-site/posts/the-rule-of-four/

**Comment by hiAndrewQuinn** [comment]
My wife called this the "secret third thing" I'm so devastated
